src/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -110,7 +110,6 @@ public void Equals(Assembly assembly1, Assembly assembly2, bool expected)
         [InlineData(typeof(AssemblyTests), true)]
         [InlineData(typeof(AssemblyPublicClass.PublicNestedClass), true)]
         [InlineData(typeof(PublicEnum), true)]
-        [InlineData(typeof(BaseClass), true)]
```

Comment:
The type no longer exists - we have enough tests for ExportedTypes so I'm not concerned about this getting removed